### PR TITLE
Settable pipeline loglevel

### PR DESCRIPTION
**Pull request by kiyo-masui**

### CODE REVIEW

**Comment by jrs65** [comment]
Looks good. I usually set the logging using the `draco.core.task.SetMPILogging` but that only works for the things derived from `MPITask`.

File: scripts/caput-pipeline [hunk 1]
```diff
@@ -13,8 +13,12 @@ products = None
 
 
 @arg('configfile', help='Configuration file to run.')
-def run(configfile):
+@arg('--loglevel', help='Logging level (DEBUG, INFO, WARNING, ERROR, CRITICAL)')
+def run(configfile, loglevel='INFO'):
     from caput.pipeline import Manager
+    import logging
+    level = getattr(logging, loglevel.upper())
+    logging.basicConfig(level=logging.INFO)
```
Comment (jrs65) [inline scripts/caput-pipeline]: Doesn't this just always set the log level to `INFO`?

Comment (kiyo-masui) [reply scripts/caput-pipeline]: I must have been drunk when I wrote this. I'll fix it.

**Comment by kiyo-masui** [comment]
Okay, how's this?

**Pull request merged**